backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/MetadataUtils.java

```diff
@@ -249,16 +254,43 @@ private static void createRowLevelPermissions(DSLContext jooq, org.jooq.Table ta
 
   protected static void saveSchemaMetadata(DSLContext sql, SchemaMetadata schema) {
     try {
-      sql.insertInto(SCHEMA_METADATA).columns(TABLE_SCHEMA).values(schema.getName()).execute();
+      String description = Objects.isNull(schema.getDescription()) ? "" : schema.getDescription();
+      sql.insertInto(SCHEMA_METADATA)
+          .columns(TABLE_SCHEMA, SCHEMA_DESCRIPTION)
+          .values(schema.getName(), description)
+          .execute();
     } catch (Exception e) {
       throw new MolgenisException("save of schema metadata failed", e);
     }
   }
 
+  protected static void updateSchemaMetadata(DSLContext sql, SchemaMetadata schema) {
+    try {
+      sql.update(SCHEMA_METADATA)
+          .set(SCHEMA_DESCRIPTION, schema.getDescription())
```

Comment:
would here then have expected schema.getMetadata().getDescription()

Reply:
?? desc is part of SchemaMetadata , var schema, should probably be renamed to schemaMetaData